app/assets/main/react_components/dashboard/climate_actions/manage_actions/dropdown-kanban/MobileKanbanContainer.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+
+const MobileKanbanContainer = ({ climateActions, children }) => {
+  const getTotalAmountOfAcceptedActionsForUser = () => {
+    let amountActionsAccepted = 0;
+    climateActions.map((action) => {
+      if (action.accepted === true) {
+        amountActionsAccepted++;
+      }
+    });
+    return amountActionsAccepted;
```

Comment:
Minor details. Perhaps use the names actionsAccepted and getAcceptedActionsForUser to reduce the name-length? :)